ftruncate: : No such process
[2019-12-13 15:04:02] INFO  (logfile_rotation_job) : log file rotation job done
[2019-12-13 15:04:02] INFO  (main) Starting MEA-EDOMUS 0.0-lite
{
	"MEAPATH":	"/Users/patrice/Developpement/Logiciels/mea-edomus-lite/demo",
	"HTML_PATH":	"./html",
	"LOGPATH":	"/Users/patrice/Developpement/Logiciels/mea-edomus-lite/demo/var/log",
	"PLUGINPATH":	"/Users/patrice/Developpement/Logiciels/mea-edomus-lite/demo/lib/mea-plugins",
	"DRIVERSPATH":	"/Users/patrice/Developpement/Logiciels/mea-edomus-lite/build/Debug",
	"XPL_VENDORID":	"mea",
	"XPL_DEVICEID":	"edomus",
	"XPL_INSTANCEID":	"demo",
	"VERBOSELEVEL":	"1",
	"HTTP_PORT":	"8083",
	"INTERFACE":	"en0",
	"RULESFILE":	"/Users/patrice/Developpement/Logiciels/mea-edomus-lite/demo/lib/mea-rules/automator.rules",
	"USERSFILE":	"users.json",
	"RULESFILESPATH":	"/Users/patrice/Developpement/Logiciels/mea-edomus-lite/demo/lib/mea-rules"
}
DATATIMESERVER : da62000
MAIN : 196dc0
[2019-12-13 15:04:02] INFO  (_timeServer_thread) : Traitement jour
start(0) : 08:35
end(0) :  16:53
start(1) : 07:58
end(1) :  17:30
[2019-12-13 15:04:02] INFO  (start_httpServer) : GUISERVER launched successfully.
PYTHONPLUGINSERVER : f4fe000
[2019-12-13 15:04:02] INFO  (start_pythonPluginServer) : PYTHONPLUGINSERVER launched successfully.
[2019-12-13 15:04:03] INFO  (start_xPLServer) : XPLSERVER launched successfully.
AUTOMATORSERVER : f604000
[2019-12-13 15:04:03] INFO  (start_automatorServer) : AUTOMATORSERVER launched successfully.
INPUTS TABLE :
--------------
rule STARTUP: (string)N/A (6)
rule NOW: (string)N/A (6)
rule FILTRE1: (string)N/A (6)
rule FILTRE2: (string)N/A (6)
rule FILTRE3: (string)N/A (6)
rule FILTRE4: (string)N/A (6)
rule I1: (string)N/A (6)
rule NEXT: (string)N/A (6)
rule O1: (string)N/A (6)
rule TIMER1: (string)N/A (6)
--------------
[2019-12-13 15:04:03] ERROR (start_interface_type_003) : enocean_init - Unable to open serial port (/dev/tty.usbserial-FTY1A5FM).
[2019-12-13 15:04:03] ERROR (start_interface_type_002) : init_xbee - Unable to open serial port (/dev/ttyUSB1).
FREE start_interface_type_002 1476
[2019-12-13 15:04:03] INFO  (mea_xPLConnectHub) : xPL Starting in Hub mode on port 53571
[2019-12-13 15:04:03] INFO  (mea_xPLConnectBroadcast) : auto-assigning IP address of 172.16.15.4
[2019-12-13 15:04:03] DEBUG (mea_xPLConnectBroadcast) : NETMASK=feffff
[2019-12-13 15:04:03] ERROR (mea_xPLConnectBroadcast) : assigned xPL Broadcast address of 172.16.15.255, port 3865
[2019-12-13 15:04:03] INFO  (start_interface_type_010) : I010 launched successfully.
[[2019-12-13 15:04:03] INFO  (main) : MEA-EDOMUS 0.0-lite starded
2019-12-13 15:04:03] INFO  (mea_init) - lancement mea_init
[2019-12-13 15:04:03] DEBUG (mea_call_python_function2) : Result of call : 1
[2019-12-13 15:04:03] INFO  (automator_timerCtrl) : start timer timer1
[2019-12-13 15:04:08] INFO  (_automator_thread) : timer wakeup signal
[2019-12-13 15:04:08] INFO  (automator_timerCtrl) : start timer timer1
[2019-12-13 15:04:13] INFO  (_automator_thread) : timer wakeup signal
[2019-12-13 15:04:13] INFO  (automator_timerCtrl) : start timer timer1
[2019-12-13 15:04:18] INFO  (_automator_thread) : timer wakeup signal
[2019-12-13 15:04:18] INFO  (automator_timerCtrl) : start timer timer1
[2019-12-13 15:04:23] INFO  (_automator_thread) : timer wakeup signal
[2019-12-13 15:04:23] INFO  (automator_timerCtrl) : start timer timer1
[2019-12-13 15:04:28] INFO  (_automator_thread) : timer wakeup signal
[2019-12-13 15:04:28] INFO  (automator_timerCtrl) : start timer timer1
[2019-12-13 15:04:33] INFO  (_automator_thread) : timer wakeup signal
[2019-12-13 15:04:33] INFO  (automator_timerCtrl) : start timer timer1
[2019-12-13 15:04:34] ERROR (xPLServer_thread) : no xPL message since 30 seconds, but we should have one ... I send me three or more !
[2019-12-13 15:04:34] ERROR (xPLServer_thread) : watchdog_recovery forced ...
[2019-12-13 15:04:34] INFO  (xPLServer_thread) : XPLSERVER thread is running
[2019-12-13 15:04:35] INFO  (_managed_processes_processes_check_heartbeats) : watchdog process for XPLSERVER
[2019-12-13 15:04:35] INFO  (_managed_processes_processes_check_heartbeats) : watchdog recovery started for XPLSERVER
[2019-12-13 15:04:35] DEBUG (stop_xPLServer) : XPLSERVER, end after 2 loop(s)
[2019-12-13 15:04:35] INFO  (stop_xPLServer) : XPLSERVER stopped successfully.
[2019-12-13 15:04:35] INFO  (start_xPLServer) : XPLSERVER launched successfully.
[2019-12-13 15:04:35] INFO  (_managed_processes_processes_check_heartbeats) : watchdog recovery done for XPLSERVER (ret=0)
[2019-12-13 15:04:35] INFO  (mea_xPLConnectHub) : xPL Starting in Hub mode on port 65289
[2019-12-13 15:04:35] INFO  (mea_xPLConnectBroadcast) : auto-assigning IP address of 172.16.15.4
[2019-12-13 15:04:35] DEBUG (mea_xPLConnectBroadcast) : NETMASK=feffff
[2019-12-13 15:04:35] ERROR (mea_xPLConnectBroadcast) : assigned xPL Broadcast address of 172.16.15.255, port 3865
[2019-12-13 15:05:09] INFO  (logfile_rotation_job) : log file rotation job start
